.vscode/launch.json

```diff
@@ -71,7 +71,6 @@
                 "SKIP_SERVICE_VERSION_REQUIREMENTS": "1",
                 "PRINT_SQL": "1",
                 "REPLAY_EVENTS_NEW_CONSUMER_RATIO": "1.0",
-                "AUTO_LOGIN": "True",
```

Comment:
This was a solution from Bianca that we agreed to revert. Did it here as I needed to test the logged out situation a lot